handlers/no_modify_css_tests/__init__.py

```diff
@@ -0,0 +1,16 @@
+from eventhandler import EventHandler
+
+NO_MODIFY_CSS_TESTS_MSG = '''This pull request modifies the contents of
+'tests/wpt/css-tests/', which are overwriten occasionally whenever the
+directory is synced from upstream.'''
+
+class NoModifyCSSTestsHandler(EventHandler):
+    DIR_TO_CHECK = "tests/wpt/css-tests"
+
+    def on_pr_opened(self, api, payload):
+        for line in api.get_diff().split('\n'):
+            if self.DIR_TO_CHECK in line:
```

Comment:
Let's make a change like https://github.com/servo/highfive/commit/1c17b665e740b0eaff0e9083e4a6fd695eb93474 and ensure we're only checking the lines that start with `diff --git`.